volatility3/framework/plugins/linux/capabilities.py

```diff
@@ -0,0 +1,207 @@
+# This file is Copyright 2023 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+from dataclasses import dataclass, astuple, fields
+from typing import Iterable, List, Tuple, Dict
+
+from volatility3.framework import interfaces, renderers, exceptions
+from volatility3.framework.constants.linux import CAP_FULL
+from volatility3.framework.configuration import requirements
+from volatility3.framework.interfaces import plugins
+from volatility3.framework.objects import utility
+from volatility3.framework.symbols.linux import extensions
+from volatility3.plugins.linux import pslist
+
+vollog = logging.getLogger(__name__)
+
+
+@dataclass
```

Comment:
You're right, dataclasses are cool!  I very much approve!  5:D

Reply:
And its members are typed, so we can also use in combination with the typing module.